client/signup/config/flows-pure.js

```diff
@@ -119,6 +119,15 @@ export function generateFlows( {
 			lastModified: '2020-12-10',
 			showRecaptcha: true,
 		},
+		{
+			name: 'onboarding-pm',
+			steps: [ 'user-pm', 'domains-pm', 'plans-pm' ],
```

Comment:
I think it would be better to keep the same steps, until they change, and then split them out. Doing so will reduce this PR to just introducing the new flow here, and we can iterate and make these steps as they're needed.